lib/wrapSql.js

```diff
@@ -1,4 +1,4 @@
-var _ = require("lodash");
+var _ = require('lodash');
 // var logger = require("node-tech-logger");
```

Comment:
Not sure this is necessary, we're only migrating code that would change on save when updating a file with prettier to avoid painful diffs in the future.